youslist.txt

```diff
@@ -2186,6 +2186,7 @@
 ||torrentpi*.com/data/file/ad/
 ||torrentpi*.com/data/file/ads/
 ||torrentsee*.com/js/jquery.ksad.js^
+||toss.im/pages/ekdil/tragoudo^
```

Comment:
If this is used only in `toss.im`, can you add `$domain=toss.im` to this line?

Reply:
I didn't check other sites but it should be ok.